src/main/java/com/bullhorn/dataloader/service/csv/CsvFileWriter.java

```diff
@@ -100,7 +94,8 @@ public synchronized void writeRow(JsonRow jsonRow, Result result) throws IOExcep
 
     public synchronized void writeAttachmentRow(String[] data, Result result) throws IOException {
```

Comment:
Suggest renaming this method to `writeRow` that differs from the `writeRow` above only by the first parameter.  Then, the above method can just call `writeRow(jsonRow.getValues())` and we can remove the redundant code.